diagrams-svg.cabal

```diff
@@ -61,6 +62,7 @@ Library
                      , lens                 >= 4.0   && < 4.12
                      , hashable             >= 1.1   && < 1.3
                      , optparse-applicative >= 0.10  && < 0.12
+                     , semigroups
```

Comment:
[packdeps](http://hackage.haskell.org/package/packdeps-0.4.0.3) and [Stackage](https://www.stackage.org/) both alert us when newer versions are available.  We're working on making the actual release have fewer manual steps.